book/scripts.md

```diff
@@ -1239,3 +1242,11 @@ Implement this behavior for `innerHTML` as a getter. Also implement
 element itself, not just its children.
 
 [innerHTML]: https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML
+
+*Script-added scripts and style sheets*: the `innerHTML` API could cause
+`<script>` or `<link>`  elements to be added to the document, but currently
+our browser does not load them when this happens. Implement this behavior.
```

Comment:
```suggestion
our browser does not load them when this happens. Fix this.
```

Also, does this actually happen in real browsers? I know that creating script elements with `document.createElement` or `document.write` works; does `innerHTML`? I somehow seem to think it doesn't, but that sounds crazy.